### DISCUSSION TITLE
Request Approved, 8 days have passed and still haven't recieved any email or the benefits

### DISCUSSION THREAD

**Discussion by alimustafa-poder** [community/community#78163] [GitHub Education] [answered]
### Hi everyone,

I submitted an application on Dec 2nd, 2023 and my request was approved. I see this on my account: "Congrats! Your request for <username and link retracted> was approved and your benefits will be available soon." I can't submit another application because it seems like there is an interval between applications. This is what I see at the top of application: "Hi, <username>! You were last verified as a student on Dec 2, 2023. It is not necessary for you to reverify at this time. There may be a wait period between verification and access to academic benefits." What do I need to do? What are the steps forward?

**Comment by adoleguizi** [comment]
me too...

**Reply by gillamir86** [reply]
Hi al
shows me your uploaded proof has already been used previously, can someone guide me how to cancel/delete previous one

**Comment by alimustafa-poder** [comment, marked as answer]
I got the benefits today. Almost after 8 days.

**Comment by fcholak** [comment]
I am still waiting... why it takes so long to get the benefits...

**Comment by Eikosa** [comment]
I'm waiting too...

**Comment by zhyemqww** [comment]
Merry Christmas! The benefits will be delayed because of the holiday.

**Comment by wilzabeast** [comment]
Have you checked out what they give you in the student pack? I'm waiting too, but the sheer amount of bonuses may be why it takes so long. I am so excited. I do not even know what at least a 1/4 of the resources are even, but they sound great. I did this for the AI, that's it...happy if that's all I got, but wow...it sure isnt.

**Comment by ShalomGH** [comment]
I'm waiting too
